engine/release-notes/24.0.md

```diff
@@ -0,0 +1,73 @@
+* libnetwork: Remove usage of the `xt_u32` kernel module from encrypted Swarm overlay networks. [moby/moby#45281](https://github.com/moby/moby/pull/45281)
+* Remove support for buildkit's deprecated `buildinfo` in favor of standard provenance attestations. [moby/moby#45097](https://github.com/moby/moby/pull/45097)
+* Remove the deprecated AUFS and legacy `overlay` storage drivers. [moby/moby#45342](https://github.com/moby/moby/pull/45342), [moby/moby#](https://github.com/moby/moby/pull/45359)
+* Remove the deprecated `overlay2.override_kernel_check` storage driver option. [moby/moby#45368](https://github.com/moby/moby/pull/45368)
+* Remove workarounds for obsolete versions of `apparmor_parser` from the AppArmor profiles. [moby/moby#45500](https://github.com/moby/moby/pull/45500)
+* API:  `GET /images/json` no longer represents empty RepoTags and RepoDiegsts as`<none>:<none>`/`<none>@<none>`. Empty arrays are be returned instead on API >= 1.43. [moby/moby#45068](https://github.com/moby/moby/pull/45068)
+
+### Deprecated
+* Deprecate the `--oom-score-adjust` daemon option. [moby/moby#45315](https://github.com/moby/moby/pull/45315)
+* API: Deprecate the `VirtualSize` field in `GET /images/json` and `GET /images/{id}/json`. [moby/moby#45346](https://github.com/moby/moby/pull/45346)
+
+### Bug fixes and enhancements
+
+* The `docker stack` command no longer validates the `build` section of Compose files. [docker/cli#4214](https://github.com/docker/cli/pull/4214)
+* Fix lingering healthcheck processes after timeout is reached. [moby/moby#43739](https://github.com/moby/moby/pull/43739)
+* Reduce the overhead of container startup when using the `overlay2` storage driver. [moby/moby#44285](https://github.com/moby/moby/pull/44285)
+* API: Handle multiple `before=` and `since=` filters in `GET /images`. [moby/moby#44503](https://github.com/moby/moby/pull/44503)
+* Fix numerous bugs in the embedded DNS resolver implementation used by user-defined networks. [moby/moby#44664](https://github.com/moby/moby/pull/44664)
+* Add `execDuration` field to the map of event attributes. [moby/moby#45494](https://github.com/moby/moby/pull/45494)
+* Swarm-level networks can now be created with the Windows `internal`, `l2bridge`, and `nat` drivers. [moby/swarmkit#3121](https://github.com/moby/swarmkit/pull/3121), [moby/moby#45291](https://github.com/moby/moby/pull/45291)
+
+### Packaging updates
+
+* Update Go to `1.20.4`. [docker/cli#4253](https://github.com/docker/cli/pull/4253), [moby/moby#45456](https://github.com/moby/moby/pull/45456), [docker/docker-ce-packaging#888](https://github.com/docker/docker-ce-packaging/pull/888)
+* Update `containerd` to [`v1.7.1`](https://github.com/containerd/containerd/releases/tag/v1.7.1). [moby/moby#45537](https://github.com/moby/moby/pull/45537)
+* Update `buildkit` to [`v0.11.4`](https://github.com/moby/buildkit/releases/v0.11.4). [moby/moby#45109](https://github.com/moby/moby/pull/45109)
```

Comment:
Should be 0.11.6: https://github.com/moby/moby/blob/1331b8c39a601c77d678f3e03f3f39cb07aa5f76/vendor.mod#L59
* https://github.com/moby/moby/pull/45214
* https://github.com/moby/moby/pull/45367